File: decompiled/sources/com/hjq/http/lifecycle/ApplicationLifecycle.java
package com.hjq.http.lifecycle;

import androidx.lifecycle.Lifecycle;
import androidx.lifecycle.LifecycleOwner;
import androidx.lifecycle.LifecycleRegistry;

public final class ApplicationLifecycle implements LifecycleOwner {
    private final LifecycleRegistry mLifecycle = new LifecycleRegistry(this);

    public Lifecycle getLifecycle() {
        return this.mLifecycle;
    }
}

File: decompiled/sources/com/hjq/http/lifecycle/ApplicationLifecycle.kt
package com.hjq.http.lifecycle

import androidx.lifecycle.Lifecycle
import androidx.lifecycle.LifecycleOwner
import androidx.lifecycle.LifecycleRegistry

class ApplicationLifecycle : LifecycleOwner {
    private val mLifecycle: LifecycleRegistry = LifecycleRegistry(this)

    override fun getLifecycle(): Lifecycle {
        return mLifecycle
    }
}
